pal/inc/rt/palrt.h

```diff
@@ -912,7 +912,7 @@ inline int __cdecl _scwprintf_unsafe(const WCHAR *_Format, ...)
 
 inline int __cdecl _vsnwprintf_unsafe(WCHAR *_Dst, size_t _SizeInWords, size_t _Count, const WCHAR *_Format, va_list _ArgList)
 {
-    if (_Count == _TRUNCATE) _Count = _SizeInWords - 1;
+    if (_Count == _TRUNCATE) _Count = _SizeInWords;
```

Comment:
@obastemur Can you review the change in PAL ?
I encountered a problem where xplat and Windows builds were producing different result when truncating string with this API